t/cli/test_etcd_grpc_healthcheck.sh

```diff
@@ -34,6 +34,8 @@ custom_clean_up() {
 
 trap custom_clean_up EXIT
 
+export ETCD_ENABLE_GRPC_GATEWAY=false
```

Comment:
why set `ETCD_ENABLE_GRPC_GATEWAY=false` here

Reply:
It is used in https://github.com/apache/apisix/pull/8548/files#diff-c54ab1e62f35fd33b56cc2d14d92095ceb6dc7a652ea75403d7e09e69c4742b6